src/components/accordion/accordion.module.scss

```diff
@@ -0,0 +1,40 @@
+.item:not(:last-child) {
+    border-bottom: 1px solid rgba(118, 113, 101, 0.3);
+}
+
+.header {
+    height: 48px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    cursor: pointer;
+}
+
+.header:focus-visible {
+    box-shadow: 0 0 0 1px #ffffff, 0 0 0 3px #116dff;
+}
```

Comment:
Could user remove it using Style Panel in Codux? We have no `focus-visible` state.

Reply:
I din't know that `focus-visible` is not supported :(
But still it should be `:focus-visible`. With `:focus` you will see the focus ring on click.